docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.3"
+#
+# Compose for the development of this docker image
+#
+services:
+  database:
+    image: "postgres:9.6"
```

Comment:
shouldn't it be moved to 10?

Reply:
Eventually, definitely. I'd suggest handling that across all repositories once the upgrade docs are in place.